src/pages/en/guides/migrate-to-astro/from-wordpress.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Migrating from WordPress
+description: Tips for migrating an existing WordPress project to Astro
+layout: ~/layouts/MigrationLayout.astro
+stub: true
+framework: WordPress
+---
+
+[WordPress](https://wordpress.org) is an open-source, personal publishing system built on PHP and MySQL.
+
+
+:::tip
+You can [use WordPress as a headless CMS for your Astro project](/en/guides/cms/wordpress/). Follow our guide to use your existing WordPress content in a new Astro project.
+:::
+
+## Key Similarities between WordPress and Astro
+
+WordPress and Astro share some similarities that will help you migrate your project:
+
+- Both WordPress and Astro are ideal for content-focused sites like blogs and support writing your content in Markdown. Although the process for adding new content is different, writing in Markdown files for your Astro blog should feel familiar if you have used Markdown syntax in your WordPress editor.
```

Comment:
Is this accurate? I thought WordPress’s text editor was a rich text interface that stores HTML rather than Markdown?

Reply:
I think you can *write using Markdown syntax* in the rich text interface, which is what I was getting at. I'll double check this.